Added Test for Addition Graph Types - #39,  #61

everlybolivar opened the pull request:
Created tests that tests the data being pulled by AlphaVantage for the RSI graph and MACD graph

everlybolivar committed:
Added Testing for additional Graphs - #39, #61
Added ChartActivityTest class to test the data used for the RSI and MACD graphs.
Added RSI value test that will assert that the RSI values are for the right stock / stock ticker, and that RSI values are actually being pulled for the graph.

everlybolivar committed:
Added Testing for additional Graphs - #39, #61
Added ChartActivityTest class to test the data used for the RSI and MACD graphs.
Added RSI value test that will assert that the RSI values are for the right stock / stock ticker, and that RSI values are actually being pulled for the graph.

everlybolivar committed:
Added Testing for additional Graphs (Part 2) - #39, #61
Added a test for the data of the MACD graph. Works similarly as the RSI test committed previously.

everlybolivar committed:
Added Testing for additional Graphs (Part 2) - #39, #61
Added a test for the data of the MACD graph. Works similarly as the RSI test committed previously.

everlybolivar committed:
Refactoring
Commit mainly reimplements functionality to the RSI button - somehow lost functionality in between merges with that of master.

everlybolivar committed:
Refactoring
Commit mainly reimplements functionality to the RSI button - somehow lost functionality in between merges with that of master.

everlybolivar committed:
Refactor
Removal of a class, as testing will be incomplete by the deadline.

everlybolivar committed:
Refactor
Removal of a class, as testing will be incomplete by the deadline.

everlybolivar committed:
Merge branch 'RSIMACD_Testing' of https://github.com/Nick9500/ministocks into RSIMACD_Testing

githubRaj commented on src/test/java/nitezh/ministock/ChartDataTesting/ChartActivityTest.java:
```diff
@@ -0,0 +1,62 @@
+
+
+    private String buildRSIUrl(String symbol) {
+        String RSIurl = "https://www.alphavantage.co/query?function=RSI&symbol=" + symbol + "&interval=daily&time_period=14&series_type=close&apikey=ZKD8M6L9CEQAK89H";
```
Instead of adding the api key in the url, append BuildConfig.AplhaVantageSecKey.

githubRaj commented on src/test/java/nitezh/ministock/ChartDataTesting/ChartActivityTest.java:
```diff
@@ -0,0 +1,62 @@
+    }
+
+    private String buildMACDURL(String symbol) {
+        String MACDurl = "https://www.alphavantage.co/query?function=MACD&symbol=" + symbol + "&interval=weekly&series_type=close&apikey=ZKD8M6L9CEQAK89H";
```
Instead of adding the api key in the url, append BuildConfig.AplhaVantageSecKey.

everlybolivar committed:
Refactor #80 - Accordance to reviewer
Changed the MACD/RSI urls so that the API key is no longer hardcoded into the link.

everlybolivar committed:
Fixed Typo #80

githubRaj reviewed (approved):
Nice
